### PR TITLE
docs: add workspace cleanup docs

### PR DESCRIPTION
**Pull request by sreya**

### CODE REVIEW

**Comment by matifali** [comment]
@bpmct @stirby we should get this in.

**Review by matifali — APPROVED** [review]

**Review by matifali — CHANGES_REQUESTED** [review]
It needs to be updated to remove all reference to the feature being experimental as we are moving it out of experiments in #11427

**Review by matifali — APPROVED** [review]

File: docs/images/template-scheduling.png
Comment (matifali) [inline docs/images/template-scheduling.png:1]: @stirby could you update screenshot as it's no more alpha.

**Review by matifali — APPROVED** [review]

File: docs/images/template-scheduling.png
Comment (matifali) [reply docs/images/template-scheduling.png:1]: @sreya we needed to update the screenshot to remove Alpha labels.

Comment (sreya) [reply docs/images/template-scheduling.png:1]: We'd need to remove the tags in the product first since they're still there.

**Pull request merged**